app/src/main/report/components/Reports/ReportDownloads.tsx

```diff
@@ -23,10 +22,9 @@ export const ReportDownloadsComponent: React.FunctionComponent<ReportDownloadsPr
     return <div className={"pl-3 pl-md-0 mb-5"}>
         <ReportTitle versionDetails={props.versionDetails}/>
         <ArtefactsSection report={props.report} versionDetails={props.versionDetails}/>
-        <DataLinks {...props.versionDetails.hash_data} />
+        <DataLinks {...props.versionDetails.data_hashes} />
         <ResourceLinks resources={props.versionDetails.resources} report={props.report}
                        version={props.versionDetails.id}/>
-        <ParameterList {...props.versionDetails.parameters} />
```

Comment:
I've gone ahead and just removed this `ParameterList` component for now, as deciding how to represent parameters in the portal is really a bigger story that we have yet to work on.